strobogrammatic.py

```diff
@@ -1,23 +1,29 @@
 import unittest
 
+
 class solution:
+
+
 	def is_strobogrammatic(self, num):
-		turn_map = {'9':'6', '6': '9', '1':'1', '8':'8'}
+		_TURN_MAP = {'9':'6', '6':'9', '1':'1', '8':'8', '0':'0'}
```

Comment:
This should be placed before the method:
class solution:
   _TURN_MAP = {}

   def is_stro...(self, num):